app/main.py

```diff
@@ -4,4 +1,47 @@
+import os
+import requests
+
+
+WEATHER_API_URL = "http://api.weatherapi.com/v1/current.json"
+DEFAULT_CITY = "Paris"
+
+
+def fetch_weather_data(api_key: str, city: str) -> dict:
+    try:
+        response = requests.get(
+            WEATHER_API_URL, params={"key": api_key, "q": city}
+        )
+        response.raise_for_status()
+        return response.json()
+    except requests.exceptions.RequestException as e:
+        print(f"Network error: {e}")
+        return {}
+
+
+def display_weather_info(data: dict) -> None:
+    if data:
+        location_time = data["location"]["localtime"]
+        temperature = data["current"]["temp_c"]
+        condition_text = data["current"]["condition"]["text"]
+        location = data["location"]["name"]
+        country = data["location"]["country"]
+        print(
+            f"{location}/{country} {location_time} Weather: "
+            f"{temperature} Celsius, {condition_text}"
+        )
+    else:
+        print("Failed to retrieve weather information.")
```

Comment:
The same applies here. Instead of printing a message, consider logging it. This way, you can have a more detailed record of the application's behavior.